source/virtualBuffers/__init__.py

```diff
@@ -366,7 +374,19 @@ def loadBuffer(self):
 
 	def _loadBuffer(self):
 		try:
-			self.VBufHandle=NVDAHelper.localLib.VBuf_createBuffer(self.rootNVDAObject.appModule.helperLocalBindingHandle,self.rootDocHandle,self.rootID,unicode(self.backendName))
+  			weblink=self.documentConstantIdentifier

```

Comment:
Possibly move all this config fetching code into its own method which just returns  the serialized labels string ready to pass to createBuffer.